### DISCUSSION TITLE
Add an option for custom file storage path on MacOS

### DISCUSSION THREAD

**Discussion by Br4thering** [LeoKlaus/Paperparrot#140] [Feature Requests]
On MacOs in a serverless usage case, I'd like to be able to configure a custom root folder for document storage (e.g. to store files under ~/documents.

In addition the support for something like the ngx PAPERLESS_FILENAME_FORMAT would be great. So that below the custom path. file can be stored under e.g. {correspondent}/{created year}/{document_type}/created.title

**Comment by LeoKlaus** [comment]
I like this. I'll see what I can do. Given that Paperparrot runs via Mac Catalyst, it doesn't necessarily have the same capabilities "native" Mac apps have, but something like this should be possible.
It will take a while to implement this, though.

**Reply by Br4thering** [reply]
No Problem, take your time.
